static/app/components/searchSyntax/parser.tsx

```diff
@@ -877,7 +878,8 @@ export class TokenConverter {
     if (
       this.config.validateKeys &&
       this.config.supportedTags &&
-      !this.config.supportedTags[getKeyName(key)]
+      !this.config.supportedTags[getKeyName(key)] &&
+      !isKnownAttribute({key: key.text, name: getKeyName(key)})
```

Comment:
Rather than hardcoding this, since this will uniformly show every `sentry.` as blue, I think we should be passing either this function or the `stringAttributes` and `numberAttributes` from `TraceItemSearchQueryBuilder` so it makes it way here via `parseSearch`